README.md

```diff
@@ -1,2 +1,55 @@
-# orocommerce-shipping-estimator
-A bundle for OroCommerce that adds a shipping estimator component.
+Aligent OroCommerce Shipping Estimator Bundle
+==============================
+This bundle adds the Shipping Estimator functionality on the OroCommerce storefront shopping list.
+
+<img src="src/Aligent/ShippingEstimatorBundle/Resources/doc/img/feature.png" alt="Shipping Estimator functionality">
```

Comment:
So this should be changed from:
`<img src="src/Aligent/ShippingEstimatorBundle/Resources/doc/img/feature.png" alt="Shipping Estimator functionality">`
to:
`<img src="src/Resources/doc/img/feature.png" alt="Shipping Estimator functionality">`
to match the path to the image from the repository root.